ts/features/cieLogin/components/CieIdLoginWebView.tsx

```diff
@@ -65,13 +75,56 @@ const CieIdLoginWebView = ({ spidLevel, isUat }: CieIdLoginProps) => {
   const [authenticatedUrl, setAuthenticatedUrl] = useState<string | null>(null);
   const loggedInAuth = useIOSelector(loggedInAuthSelector, _isEqual);
   const loginUri = getCieIDLoginUri(spidLevel, isUat);
+  const [isLoadingWebView, setIsLoadingWebView] = useState(false);
 
   const navigateToCieIdAuthenticationError = useCallback(() => {
     navigation.replace(ROUTES.AUTHENTICATION, {
       screen: ROUTES.AUTHENTICATION_CIE_ID_ERROR
     });
   }, [navigation]);
 
+  const navigateToCieIdAuthUrlError = useCallback(
+    (url: string) => {
+      navigation.replace(ROUTES.AUTHENTICATION, {
+        screen: ROUTES.AUTHENTICATION_CIE_ID_INCORRECT_URL,
+        params: { url }
+      });
+    },
+    [navigation]
+  );
+
+  const navigateToLandingScreen = useCallback(() => {
+    navigation.navigate(ROUTES.AUTHENTICATION, {
+      screen: ROUTES.AUTHENTICATION_LANDING
+    });
+  }, [navigation]);
+
+  const checkIfUrlIsWhitelisted = useCallback(
+    (url: string) => {
+      const whitelistedUrls = [
+        "https://idserver.servizicie.interno.gov.it",
+        "https://oidc.idserver.servizicie.interno.gov.it",
+        "https://mtls.oidc.idserver.servizicie.interno.gov.it",
+        "https://mtls.idserver.servizicie.interno.gov.it",
+        "https://ios.idserver.servizicie.interno.gov.it",
+        "https://ios.oidc.idserver.servizicie.interno.gov.it"
+      ];
```

Comment:
Since this list contains constants values, what do you think about moving it outside the component?